core/src/bin/validate.ts

```diff
@@ -78,7 +78,9 @@ export class Validate extends CLI {
         throw new Error("validate-rollback");
       });
     } catch (error) {
-      if (error.message !== "validate-rollback") throw error;
+      if (error.message !== "validate-rollback") {
+        GrouparooCLI.logger.fatal(`Validation failed - ${error.message}`);
```

Comment:
Note: `GrouparooCLI.logger.fatal` does call `process.exit`, so it does have the same effect as the old "throw" of ending the process.

Reply:
And with a non-zero exit code.